domainlab/utils/generate_benchmark_plots.py

```diff
@@ -12,10 +12,13 @@
 matplotlib.use('Agg')
 
 COLNAME_ALGO = "algo"
+COLNAME_IDX_PARAM = "param_index"
 COLNAME_PARAM = "params"
+G_DF_PLOT_COL_START = 2   # first two columns are ['param_index','task',
+G_DF_PLOT_COL_METRIC_START = 7   # first 0-6 columns are not metric
 
```

Comment:
Could you add one line comment copying the header of our current CSV? SO THE definition of the Macro will make more sense.